screen-capture/getdisplaymedia.https.html

```diff
@@ -18,9 +17,7 @@
 const j = obj => JSON.stringify(obj);
 
 async function getDisplayMedia(constraints) {
-  const p = new Promise(r => button.onclick = r);
-  await test_driver.click(button);
-  await p;
+  await test_driver.bless("getDisplayMedia", () => navigator.mediaDevices.getDisplayMedia(constraints));
```

Comment:
Oops, my suggestion didn't make sense on its own, you'd also need to remove the following line. (But see other comment about probably removing this helper altogether.)